contracts/ERC721CM.sol

```diff
@@ -336,29 +328,12 @@ contract ERC721CM is IERC721M, ERC721ACQueryable, Ownable, ReentrancyGuard {
      * @dev Mints token(s).
      *
      * qty - number of tokens to mint
-     * proof - the merkle proof generated on client side. This applies if using whitelist.
-     * timestamp - the current timestamp
-     * signature - the signature from cosigner if using cosigner.
-     */
-    function mint(
```

Comment:
for existing mints, how does the client know which `mint` function to use? given that the function signature changed.